Week3/homework/ToDoApp_dump.sql

```diff
@@ -0,0 +1,113 @@
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+DROP SCHEMA IF EXIST ToDoApp;
+CREATE SCHEMA ToDoApp;
+USE ToDoApp;
+SET AUTOCOMMIT=0;
+
+--
+-- Table structure for table `todolists`
+--
+
+DROP TABLE IF EXISTS `todolists`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+ SET character_set_client = utf8mb4 ;
+CREATE TABLE `todolists` (
+  `User_Id` varchar(50) NOT NULL,
+  `ToDoList_Id` varchar(50) NOT NULL,
+  `ToDoList_Name` varchar(50) NOT NULL,
+  `Reminder` datetime DEFAULT NULL,
+  PRIMARY KEY (`ToDoList_Id`),
+  FOREIGN KEY (`User_Id`) REFERENCES `users`(`User_Id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `todolists`
+--
+
+LOCK TABLES `todolists` WRITE;
+/*!40000 ALTER TABLE `todolists` DISABLE KEYS */;
+INSERT INTO `todolists` VALUES ('RfVkWYLz7t9xWnECw8zfo','-gEwx6en9qLEr0ZRuE5Q-','cleaning material','2019-11-03 00:00:00'),('Hk0yPR8DvmIC7WYOvDkgP','0U8YAoKhi31TGu9BHPYjt','furniture','2019-10-02 00:00:00'),('Vn9vc1d5XFxE4T4B0PVGe','3UthG7gWxIAcI-WbXAM9j','grocery1','2019-05-29 00:00:00'),('ySEt2NHE0HvW0sAtX97mK','6oWRF5tlAh234uyPUHn76','house stuff 123','2019-07-09 00:00:00'),('8vS4PDKvV-aXUgnSu1I4B','BvDSlKNJeadmhjtyANlBG','library','2019-04-02 00:00:00'),('Vn9vc1d5XFxE4T4B0PVGe','iLI-Z83ooba6RQAglkwyk','laundry','2019-05-02 00:00:00'),('ySEt2NHE0HvW0sAtX97mK','ITGAUPXcvRQJRTHVuZYyJ','shopping','2019-07-09 00:00:00'),('KHuSxKyVy6AGwuIW3-o1y','Lr8AxOBnJ6U_j7XiAl4Vq','fruits','2019-10-02 00:00:00'),('sPcDYztwcEjYTEIYLP8h3','lZSG7Ww5h84e9J5LOxe1T','digital stuff','2019-10-02 00:00:00'),('HCN4HAjIj3lIKB9tb9doc','MsOjgJCz0KuUQ9z_4NTUu','language','2019-09-02 00:00:00'),('8wuS06DCjCTqv8AD6qTUX','NEeJQhlvb7r4O3hMsd7CI','housing','2019-04-02 00:00:00'),('8wuS06DCjCTqv8AD6qTUX','PKcCDX58hzwDZAv3ETGzC','course','2019-04-02 00:00:00'),('HCN4HAjIj3lIKB9tb9doc','Pyqn2KQCP_VAJc2WUZ3Q3','digital stuff','2019-10-02 00:00:00'),('RfVkWYLz7t9xWnECw8zfo','rUv1kviNl2EWDH1lwsopq','clothing','2019-10-02 00:00:00'),('tIzOGGdf-95KfrApfy6yq','sJDNvkiIpkARhvdkC_sMt','meetings','2019-04-09 00:00:00'),('Hk0yPR8DvmIC7WYOvDkgP','SLAeE8gDpTg0S36tF4MNm','stuff for kitchen','2019-10-02 00:00:00'),('KHuSxKyVy6AGwuIW3-o1y','t9iy-ihFkcgYyhm18Nqxb','snacks','2019-10-02 00:00:00'),('8vS4PDKvV-aXUgnSu1I4B','W41mpsUE1I5FXUbJWoOql','school','2019-04-02 00:00:00');
+/*!40000 ALTER TABLE `todolists` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `todos`
+--
+
+DROP TABLE IF EXISTS `todos`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+ SET character_set_client = utf8mb4 ;
+CREATE TABLE `todos` (
+  `ToDoList_Id` varchar(50) NOT NULL,
+  `Todo_Id` varchar(50) NOT NULL,
+  `Todo_Name` varchar(100) NOT NULL,
+  `Done` enum('true','false') DEFAULT NULL,
```

Comment:
**Non-blocking**

`enum` — 👌🏻

Don't you want to specify default value as one of `enum`'s values? Let's say `false`

Reply:
Done :pray: